About me

adalexander2 opened the pull request:
I don't know if I got the picture to work this time, not sure what's wrong with the image

Commit:
Adding about me page

Commit:
Adding about me page

Commit:
Merge branch 'template' of https://github.com/adalexander2/601_Spring_2023 into template

# Conflicts:
#	about/AnnieAlexander.qmd

adalexander2 committed:
Delete AnnieAlexander.qmd

SidharthSaluja reviewed (commented):
Looks like you forgot to add your image in the about/images folder. If you don’t add that the image it won’t be displayed and make sure you render before submitting.

adalexander2 committed:
Add files via upload

Commit:
about me again

adalexander2 committed:
Delete AnnieAlexanderAbout.qmd

Commit:
about me again

Merge branch 'template' of https://github.com/adalexander2/601_Spring_2023 into template

# Conflicts:
#	about/AnnieAlexander.qmd

adalexander2 committed:
Delete AnnieAlexander.qmd

adalexander2 committed:
Delete AnnieAlexanderAbout.qmd

adalexander2 committed:
Update AnnieAlexanderAboutMe.qmd
